examples/mallocMC_example01.cpp

```diff
@@ -30,12 +30,27 @@
 #include <iostream>
 #include <mallocMC/mallocMC.hpp>
 #include <numeric>
+#include <cassert>
+#include <algorithm>
 
 using Dim = alpaka::DimInt<1>;
 using Idx = std::size_t;
-// using Acc = alpaka::AccCpuThreads<Dim, Idx>;
-// using Acc = alpaka::AccCpuOmp2Threads<Dim, Idx>;
+
+#if defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
 using Acc = alpaka::AccGpuCudaRt<Dim, Idx>;
+#elif defined(ALPAKA_ACC_GPU_HIP_ENABLED)
+using Acc = alpaka::AccGpuHipRt<Dim, Idx>;
+#elif defined(ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLED)
+using Acc = alpaka::AccCpuThreads<Dim, Idx>;
+#elif defined(ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLED)
+using Acc = alpaka::AccCpuOmp2Threads<Dim, Idx>;
+#elif defined(ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED)
+using Acc = alpaka::AccCpuOmp2Blocks<Dim, Idx>;
+#elif defined(ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED)
+using Acc = alpaka::AccCpuSerial<Dim, Idx>;
+#else
+#error "Selected alpaka accelerator is not supported!"
+#endif
```

Comment:
This looks similar to the alpaka example accelerator. Could we use this one instead?

Reply:
@psychocoderHPC: Can we use `alpaka::ExampleDefaultAcc<Dim, Idx>` here?